stories/ranking.stories.jsx

```diff
@@ -0,0 +1,189 @@
+      expect(item.dataset.value).toBe('Most')
+    })
+
+    await userEvent.click(item.querySelector('#rankingNeutral'))
+    await step('"Neutral" option selectable and sets item value', async () => {
+      expect(item.dataset.value).toBe('Neutral')
+    })
+
+    await userEvent.click(item.querySelector('#rankingLeast'))
+    await step('"Least" option selectable and sets item value', async () => {
+      expect(item.dataset.value).toBe('Least')
+    })
+  })
+}
+
+const testDisabledWithDefaultNeutral = async (step, numericalIdentifier = '', defaultValue = 'Neutral') => {
+  let item
+  await common.asyncSleep(600)
+
+  await step('Item should be found', async () => {
+    item = document.querySelector(`#testRanking-${numericalIdentifier}`)
+    await step('Item DOM Node should be found', async () => {
+      expect(item).toBeTruthy()
+    })
+    await step(`Initial value should be ${defaultValue}`, async () => {
+      expect(item.dataset.value).toBe(defaultValue)
+    })
+  })
+
+  await step('When disabled, states should not be selectable and item value should not change', async () => {
+    await userEvent.click(item.querySelector('#rankingMost'))
+    await step('"Most" option not selectable; does not set item value', async () => {
+      expect(item.dataset.value).toBe(defaultValue)
+    })
+
+    await userEvent.click(item.querySelector('#rankingNeutral'))
+    await step('"Neutral" option not selectable; does not set item value', async () => {
+      expect(item.dataset.value).toBe(defaultValue)
+    })
+
+    await userEvent.click(item.querySelector('#rankingLeast'))
+    await step('"Least" option not selectable; does not set item value', async () => {
+      expect(item.dataset.value).toBe(defaultValue)
+    })
+  })
+}
+
+const generateComponent = (numericalIdentifier, disabled = false, defaultValue = '') => {
+  return (
+    <Ranking
+      disabled={disabled}
+      defaultValue={defaultValue}
+      id={`testRanking-${numericalIdentifier}`}
+      block="true"
+      large="true"
+      className={['itemComponent', 'className2', 'className3']}
+      onSelect={e => {
+        console.log(e.target.value)
+      }}
+    ></Ranking>
+  )
+}
+export const Functionality_Enabled_and_No_Default = {
+  render: () => generateComponent(1, false, ''),
+  play: ({ step }) => {
+    testEnabledNoDefault(step, '1')
+  },
+}
+
+export const Functionality_Disabled_and_No_Default = {
+  render: () => generateComponent(2, true, ''),
+  play: ({ step }) => {
+    testDisabledNoDefault(step, '2')
+  },
+}
+
+export const Functionality_Enabled_and_Neutral_Default = {
+  render: () => generateComponent(3, false, 'Neutral'),
+  play: ({ step }) => {
+    testEnabledWithDefaultNeutral(step, '3', 'Neutral')
+  },
+}
+
+export const Functionality_Disabled_and_Neutral_Default = {
+  render: () => generateComponent(4, true, 'Neutral'),
+  play: ({ step }) => {
+    testDisabledWithDefaultNeutral(step, '4', 'Neutral')
+  },
+}
```

Comment:
Please create a case where the default value is "bad value" and make sure nothing bad happens.